ntbs-service/Models/Entities/NotificationSite.cs

```diff
@@ -12,8 +12,8 @@ public class NotificationSite : ModelBase
         public int SiteId { get; set; }
         public virtual Site Site { get; set; }
 
-        [Display(Name = "Site description")]
-        [RequiredIf("SiteId == SiteId.OTHER && ShouldValidateFull", ErrorMessage = ValidationMessages.FieldRequired)]
+        [Display(Name = "Site name")]
+        [RequiredIf("ShouldValidateFull && SiteId == SiteId.OTHER", ErrorMessage = ValidationMessages.FieldRequired)]
```

Comment:
Is there a reason we're swapping this order? I'm assuming removing the `ShouldValidateFull` like we mentioned wasn't super straight-forward?